packages/taiko-client/driver/txlist_decompressor/txlist_decompressor.go

```diff
@@ -72,6 +73,16 @@ func (v *TxListDecompressor) TryDecompress(
 		return []byte{}
 	}
 
+	// Re-encode txs when txs contains invalid tx
+	for _, tx := range txs {
+		// Skip invalid L2 blob tx
+		if tx.Type() == types.BlobTxType {
```

Comment:
if tx.Type() == types.BlobTxType  && fork <= SHANGHAI ??

Reply:
We won't support l2 blob tx after cancun either.

Reply:
filter blob tx belongs to `taiko-geth` function, it seems inappropriate to decide how geth should do.

Reply:
Actually don't think this is necessary? taiko-geth will handle it?

Reply:
If taiko-geth will do this case, taiko-geth has to do it now.
It is a breaking bug and needs to be quickly fixed.